cppapi/server/dserver.cpp

```diff
@@ -484,6 +480,61 @@ void DServer::init_device()
 	}
 }
 
+void DServer::server_init_hook()
+{
+#ifdef HAS_RANGE_BASE_FOR
+    for (DeviceClass *dclass : this->get_class_list())
+    {
+        for (DeviceImpl *device : dclass->get_device_list())
+        {
+            cout4 << "Device " << device->get_name_lower() << " executes init_server_hook" << std::endl;
```

Comment:
" executes init_server_hook"  => " executes server_init_hook()" 
Same at line 518

Reply:
@bourtemb Thx. The fix was done by @Ingvord already. I've folded into the correct commit.